src/components/forms/TextInput/TextInput.tsx

```diff
@@ -18,23 +17,7 @@ interface RequiredTextInputProps {
 interface CustomTextInputProps {
   className?: string
   validationStatus?: 'error' | 'success'
-  /**
-   * @deprecated since 1.6.0, use validationStatus
-   */
-  error?: boolean
-  /**
-   * @deprecated since 1.6.0, use validationStatus
-   */
-  success?: boolean
   inputSize?: 'small' | 'medium'
```

Comment:
We actually don't use this prop at all in storybook, and USWDS doesn't show it in their examples, though it comes from the following:

<img width="1340" alt="image" src="https://user-images.githubusercontent.com/15805554/163622106-991dfae7-075a-4f8a-8a79-93a91f2be50d.png">

Wondering if we should create a `type` that enumerates those valid sizes, or just leave it up to consumers to pass in the desired `className`.

If the latter is chosen, I can remove the prop here and call it out as part of the breaking release.

Thoughts?

Reply:
I _have_ been meaning to see what it would look like to create a nice interface for `2xs`...`2xl` passed in as a `size` prop on a couple of components, which would just append `size` to the `className` like so:

```TSX
`usa-input--${size}`: size
```

I think its a nice convenience to offer, and probably doesn't involve much overhead on our part besides combing through the code. So that's what the alternative might look like.

Reply:
Oh, I like that. I think my only 🤔 was if that would make maintenance harder because we would be using dynamically calculated class names instead of having to list out all the potential options ourselves in a `classnames` object and having it readable in the code.  But thinking about it more, that's probably what storybook is for - we can show all the variants of things there.